src/sql/statement.rs

```diff
@@ -50,6 +50,53 @@ use tokio::io::AsyncBufReadExt;
 lazy_static! {
     static ref SHOW_DATABASES_DESC: RelationDesc =
         { RelationDesc::empty().add_column("Database", ScalarType::String) };
+    static ref SHOW_INDEXES_DESC: RelationDesc = RelationDesc::new(
+        RelationType::new(vec![
+            ColumnType::new(ScalarType::String),
+            ColumnType::new(ScalarType::String),
+            ColumnType::new(ScalarType::String).nullable(true),
+            ColumnType::new(ScalarType::String).nullable(true),
+            ColumnType::new(ScalarType::Bool),
+            ColumnType::new(ScalarType::Int64),
+        ]),
+        vec![
+            "Source_or_view",
+            "Key_name",
+            "Column_name",
+            "Expression",
+            "Null",
+            "Seq_in_index",
+        ]
+        .iter()
+        .map(|s| Some(*s))
+        .collect::<Vec<_>>(),
```

Comment:
Unnecessary `collect()`; since `RelationDesc` takes an `IntoIterator` you can just hand it the iterator directly.

Also, if you use `into_iter` on the vec, you won't need the dereference:

All together:

```rust
        vec![
            "Source_or_view",
            "Key_name",
            "Column_name",
            "Expression",
            "Null",
            "Seq_in_index",
        ]
        .into_iter()
        .map(Some)
```

Reply:
Done, thanks for the tip!